### PR TITLE
Add gaming contract test function

### PR DESCRIPTION
**Pull request by don1989**
## Why
- Need a way to test gaming functions and forward call 

### How
- Add a button that populates a transaction and sends it for signing

### How Has This Been Tested?
- Locally e2e

### Screenshot
Screenshot 2022-12-08 at 18 13 57

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -630,6 +634,73 @@ class App extends React.Component<any, any> {
...
+      const _gasLimit = 21000;
+      const gasLimit = sanitizeHex(convertStringToHex(_gasLimit));
+
+      const _value = 0;
+      const value = sanitizeHex(convertStringToHex(_value));
+
+      const _nonce = 0;
+      const nonce = sanitizeHex(convertStringToHex(_nonce));
+
+      const tx: ITxData = {
```
[the remark this answers is not in the document]
Comment (Alekso91) [reply src/App.tsx:671]: from: should be the EOA owner of the gamingWallet (I think from what I see you currently take the connected address)
to: has to be the gaming wallet (seems to be gamingContract at the moment)
data: seems good

[the remark this answers is not in the document]
Comment (don1989) [reply src/App.tsx:671]: @Alekso91 

Thanks, I'll change the `to` field. 

The `from` field I'm not sure we'll be able to change as we do not send both EOA and gamingWallet addresses. The dapp will only be aware of the gamingWallet address so we'll need to handle that on our side in the relay controller @avmturo is working on to change the `from` field there. Does that sound right guys?

[the remark this answers is not in the document]
Comment (Alekso91) [reply src/App.tsx:671]: Then you could also be able to put gamingWallet.owner() in the `from` field. Sounds good

[the remark this answers is not in the document]
Comment (Alekso91) [reply src/App.tsx:671]: Anyway you should have the EOA at some point as it needs to sign the TX :)

[the remark this answers is not in the document]
Comment (avmturo) [reply src/App.tsx:671]: From field is fine, in our walletConnect relay we will delete the from field and add the EOA field as part of the message intercept. There is no way for us to guarantee all DAPPS use the same method to get the gamingWallet.owner(), so better for us to deal with it on our end

**Pull request merged**